src/main/scala/zio/zmx/diagnostics/ZMXConfig.scala

```diff
@@ -16,8 +16,8 @@
 
 package zio.zmx.diagnostics
 
-case class ZMXConfig(host: String, port: Int, debug: Boolean)
+final case class ZMXConfig(host: String, port: Int, debug: Boolean)
 
 object ZMXConfig {
-  def empty = new ZMXConfig(host = "localhost", port = 1111, debug = false)
+  val empty = ZMXConfig(host = "localhost", port = 1111, debug = false)
```

Comment:
val Empty (with capital start letter) if we follow https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods